site/content/en/_index.html

```diff
@@ -56,7 +56,7 @@ <h2 class="text-center pb-3">Companies using Agones</h2>
 		<a href="http://altavr.io/"><img alt="altavr" width="100" src="images/altavr-logo.png" /></a>
 		<a href="https://www.embark.games/"><img alt="Embark Studios" width="50" src="images/embark.png" /></a>
 		<a href="http://accelbyte.io/"><img alt="accelbyte" width="100" src="images/accelbyte.png" /></a>
-		<a href="https://space.game/"><img alt="Space Game" width="80" src="images/spacegame.png" /></a>
+		<img alt="Space Game" width="80" src="images/spacegame.png" />
```

Comment:
From a brief Google search I couldn't find a new URL. We should remove this whole line, as well as the `site/static/images/spacegame.png` image since there's no URL.

```suggestion
```

Reply:
@NBoychev FYI, since this was your link!

Reply:
i also checked their [twitter](https://x.com/spacedotgame) official. but I couldn't find any new links.

Reply:
I pushed a second commit to remove everything so we can get this in ASAP as it's breaking our CI / CD test suite. They can resubmit a new PR with a new link and image later.